interface/src/app/tribe_client/directives.js

```diff
@@ -77,4 +77,19 @@ angular.module('adage.tribe_client.directives', [
     };
 }])
 
+// Directive for whole gene search form
+.directive('genesetSearchForm', function( ) {
+    return {
+        controller: ['$scope', function($scope) {
```

Comment:
Code review isn't about making more work. But if it's not clear to someone that you're planning something, they aren't going to be able to know that when they review. :)

Even better - through a `TODO:` at the front of that comment and codeclimate will flag it for you automatically.